src/sidebar/components/test/search-input-test.js

```diff
@@ -1,68 +1,88 @@
 'use strict';
 
-const angular = require('angular');
+const { createElement } = require('preact');
+const { mount } = require('enzyme');
 
-const util = require('../../directive/test/util');
+const SearchInput = require('../search-input');
 
-describe('searchInput', function() {
+describe('SearchInput', () => {
   let fakeStore;
 
-  before(function() {
-    angular
-      .module('app', [])
-      .component('searchInput', require('../search-input'));
-  });
+  const createSearchInput = (props = {}) =>
+    // `mount` rendering is used so we can get access to DOM nodes.
+    mount(<SearchInput {...props} />);
+
+  function typeQuery(wrapper, query) {
```

Comment:
This is clever! Are you just kinda figuring these out as you go, or are there cookbook-y-type resources out there?

Reply:
This is just figured out as I go. If we end up repeating something often we can make a utility out of it.